examples/test_util/EclOutput.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+   Copyright 2019 Statoil ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <iomanip>
+
+#include <examples/test_util/data/EclIOdata.hpp>
+
+namespace EIOD = Opm::ecl;
+
+
+class EclOutput
+{
+public:
+    explicit EclOutput(std::ofstream& inputFileH);
```

Comment:
I would suggest having a `bool formatted` flag in the `EclOutput` constructor - then the *only* public method in the `EclOutput` class would be `write(const std::vector<T>& array)` which would *both* handle the formatted/unformatted question and also call the (internal) implementations `writeArray()` and `writeHeader()`.

The current API allows both mixing formatted and unformatted output in one file, and also writing only headers or only data - neither makes any sense.

Reply:
done.